stdlib/http/src/main/java/org/ballerinalang/net/http/websocketclientendpoint/InitEndpoint.java

```diff
@@ -78,37 +78,36 @@ public void execute(Context context) {
 
         String remoteUrl = clientEndpointConfig.getStringField(WebSocketConstants.CLIENT_URL_CONFIG);
         Value clientServiceType = clientEndpointConfig.getTypeField(WebSocketConstants.CLIENT_SERVICE_CONFIG);
-        Service service = BLangConnectorSPIUtil.getServiceFromType(context.getProgramFile(), clientServiceType);
-        if (service == null) {
-            throw new BallerinaConnectorException("Cannot find client service: " + clientServiceType);
+        Service service = null;
+        if (clientServiceType != null) {
```

Comment:
If clientServiceType is null, the service is always null as it doesn't get assigned for any other value. Is this expected?

Reply:
Yes. I am handling the case when client service is not specified in the client endpoint in this PR so if the config is null service is going to be null. I am not creating an empty service because it can cause main method to hang if client endpoint is used in main even when not specifying a client service.